test/test_jit.py

```diff
@@ -6169,7 +6169,7 @@ def test_not_cast(x):
         with self.assertRaisesRegex(RuntimeError, r"Could not cast value of type Tuple\[Tensor, Tensor\]"):  # noqa: W605
             @torch.jit.script
             def test_mult(x, y):
-                return not(x, y)
+                return not (x, y)
```

Comment:
Is this a not as well?

Reply:
I don't follow what you mean here? This a non-functional change and is suppose to raise an error anyway.

Reply:
I wonder what that supposed to do? I.e. `not (x, y)` is always False, isn't it?